src/teleop/jetson/jetson_teleop.py

```diff
@@ -83,6 +85,10 @@ def teleop_drive_callback(self, msg: Joy):
 
 
 class ArmControl:
+
+    __arm_mode = "arm_disabled"
```

Comment:
Generally the convention for private variables in python is single underscore at the beginning, not double underscore

Reply:
Yeah I had just read about name mangling when I wrote that but I guess it doesn't really matter since the class isn't being inherited anywhere 
https://docs.python.org/3/tutorial/classes.html#private-variables

Reply:
Ah yeah I don't think we should be getting into name mangling unnecessarily